docs/for_developers.md

```diff
@@ -74,25 +74,25 @@ flowchart LR
 
     === "Linux"
         ```bash
-        CELERY_BROKER_URL=redis://localhost:6379/0 CELERY_RESULT_BACKEND=redis://localhost:6379/0 poetry run celery --app yaptide.celery.worker worker -P eventlet --loglevel=debug
+        PATH=$PATH:bin/ BACKEND_INTERNAL_URL=http://localhost:5000 CELERY_BROKER_URL=redis://localhost:6379/0 CELERY_RESULT_BACKEND=redis://localhost:6379/0 poetry run celery --app yaptide.celery.worker worker -P eventlet --loglevel=debug
         ```
 
     === "Windows (PowerShell)"
         ```powershell
-        $env:CELERY_BROKER_URL="redis://localhost:6379/0"; $env:CELERY_RESULT_BACKEND="redis://localhost:6379/0"; poetry run celery --app yaptide.celery.worker worker -P eventlet --loglevel=debug
+        $env:PATH+="\bin\"; $env:BACKEND_INTERNAL_URL=http://localhost:5000; $env:CELERY_BROKER_URL="redis://localhost:6379/0"; $env:CELERY_RESULT_BACKEND="redis://localhost:6379/0"; poetry run celery --app yaptide.celery.worker worker -P eventlet --loglevel=debug
```

Comment:
Using localhost gave me these errors:

![image](https://github.com/yaptide/yaptide/assets/7374733/62fa9803-b135-4266-a253-8598c04d7504)


switching to 127.0.0.1 solved the problems:

![image](https://github.com/yaptide/yaptide/assets/7374733/721c1fc6-4bb8-49e2-9786-7fe92492518a)
![image](https://github.com/yaptide/yaptide/assets/7374733/472e52dc-a1e7-4500-84e9-9ddbd65c5d16)